test/vtk/test_pvtr_reader.cpp

```diff
@@ -77,8 +77,8 @@ int main(int argc, char** argv) {
         expect(eq(ordinates_z.size(), std::size_t{0}));
         expect(std::ranges::is_sorted(ordinates_x));
         expect(std::ranges::is_sorted(ordinates_y));
-        std::ranges::unique(ordinates_x);
-        std::ranges::unique(ordinates_y);
+        [[maybe_unused]] const auto ret = std::ranges::unique(ordinates_x);
+        [[maybe_unused]] const auto ret = std::ranges::unique(ordinates_y);
```

Comment:
ah crap, this fails in the pipeline now because of duplicate declaration (merged w/o actions to save runner minutes)... will push a fix

Reply:
ah sorry stupid...

Reply:
I could have seen properly looked at it as well :/ I pushed it to #264 . The branch name diverged quite a bit from what is done on it, but well...